tests/utils.py

```diff
@@ -12,7 +12,7 @@
     get_module_system_for_user,
     make_track_function,
 )
-from lms.djangoapps.courseware.tests.factories import StaffFactory
+from common.djangoapps.student.tests.factories import StaffFactory
```

Comment:
I integrated this xBlock edx platform `lilac release` so it was complaining about the import there.